fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -1087,7 +1086,14 @@ public FedoraResource getResourceFromPath(final String externalPath) {
         try {
             final FedoraResource fedoraResource = resourceFactory.getResource(transaction(), fedoraId);
 
-            if (fedoraResource instanceof Tombstone) {
+            final FedoraResource originalResource;
+            if (fedoraId.isMemento()) {
+                 originalResource = fedoraResource.getOriginalResource();
+            } else {
+                originalResource = fedoraResource;
+            }
+
+            if (originalResource instanceof Tombstone) {
                 final String tombstoneUri = identifierConverter().toExternalId(
```

Comment:
I was hoping that the test entitled: "Memento of deleted resource still available" would work given this change. However, following those test steps results in:
```
HTTP/1.1 400 Bad Request
Path is invalid: /binary/fcr:versions/20200603235900/fcr:tombstone
```

Reply:
This should now be resolved. Though the test that confirms this might need some work as it was passing before the most recent change so I'm not sure it is actually useful.